src/parser/jobs/dnc/modules/DirtyDancing.tsx

```diff
@@ -289,6 +304,38 @@ export default class DirtyDancing extends Module {
 				<Plural value={driftedTechnicals} one="# Technical Step was" other="# Technical Steps were"/> lost due to drift.
 			</Trans>,
 		}))
+
+		const zeroStandards = this.danceHistory.filter(dance => dance.dirty && dance.initiatingStep.ability.guid === ACTIONS.STANDARD_STEP.id &&
```

Comment:
This is double-counting the 0 step dance within the generic "missing steps" suggestion as well.  Is there a better way to do the two suggestions so you don't wind up with the same dance generating both suggestions?  Or is it intended to generate a double suggestion?

Reply:
I think at some point I considered this, and then forgot to actually do something about it. Since these are a more important sub-category of the missing steps suggestion, I think it's fair not to double-count them. Fixed.